examples/shell/qpg6100/.gn

```diff
@@ -12,8 +12,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import("//build_overrides/chip.gni")
+
 # The location of the build configuration file.
-buildconfig = "//build/config/BUILDCONFIG.gn"
+buildconfig = rebase_path("${chip_root}/build/config/BUILDCONFIG.gn")
```

Comment:
I wonder if you should call rebase_path for platforms where you keep third_party/connectedhomeip symlink. This may change location of the output files (ABS_PATH) and shouldn't be necessary.